### PR TITLE
ref #660: Add function to (re) call the Wawi export

### PR DESCRIPTION
**Pull request by UlrichKu**
| Q             | A
| ------------- | ---
| Branch        | 7.1.x
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed issues  | chameleon-system/chameleon-system#660
| License       | MIT

### CODE REVIEW

File: src/ShopBundle/objects/TCMSTableEditor/TCMSTableEditorShopOrderEndPoint.class.php [hunk 1]
```diff
@@ -105,6 +143,20 @@ public function GetFrontendActionUrlToSendOrderEmail($sMail = null)
...
+    {
+        $translator = $this->getTranslator();
+
+        if (false !== $this->oTable->ExportOrderForWaWiHook($this->oTable->GetPaymentHandler())) {
+            $this->oTable->MarkOrderAsExportedToWaWi(true);
+
+            return $translator->trans('chameleon_system_shop.orders.export_success');
+        }
+
+        // TODO this should however be a proper error?
```
Comment (pixeljunkie) [inline src/ShopBundle/objects/TCMSTableEditor/TCMSTableEditorShopOrderEndPoint.class.php:156]: A general should be o.k. here, as long as we log errors during the export.

Comment (UlrichKu) [reply src/ShopBundle/objects/TCMSTableEditor/TCMSTableEditorShopOrderEndPoint.class.php]: Added a simple error display

File: src/ShopBundle/objects/TCMSTableEditor/TCMSTableEditorShopOrderEndPoint.class.php [hunk 2]
```diff
@@ -105,6 +144,27 @@ public function GetFrontendActionUrlToSendOrderEmail($sMail = null)
         return $sURL;
     }
 
+    public function exportOrderToWaWi(): array
```
Comment (pixeljunkie) [inline src/ShopBundle/objects/TCMSTableEditor/TCMSTableEditorShopOrderEndPoint.class.php:147]: The name should not include the german "WaWi" abbreviation even if the methods it is calling was misnamed.
We should call it exportOrders() instead. We don't know if it is exported to an inventory system or something else (could be an FTP transfer, email etc.). The ExportOrderForWaWiHook is misnamed, but we can't rename it without BC breaks.

Comment (UlrichKu) [reply src/ShopBundle/objects/TCMSTableEditor/TCMSTableEditorShopOrderEndPoint.class.php]: Renamed

**Pull request merged**